mobile-app/app/screens/AppNavigator/screens/Loans/components/AddOrRemoveCollateralForm.tsx

```diff
@@ -242,23 +255,51 @@ export const AddOrRemoveCollateralForm = memo(({ route }: Props): JSX.Element =>
         </ThemedText>
       </InputHelperText>
       <ScrollView
-        horizontal contentContainerStyle={tailwind(['flex justify-between flex-row', {
+        horizontal contentContainerStyle={tailwind(['flex justify-between items-center flex-row', {
         'flex-grow h-7': Platform.OS !== 'web',
         'w-full': Platform.OS === 'web'
       }])}
       >
+        <ThemedText style={tailwind('mr-2')}>{translate('components/AddOrRemoveCollateralForm', 'Vault %')}</ThemedText>
+        <ThemedView
+          style={tailwind('items-center flex flex-row py-1 px-1.5 rounded-2xl')}
+        >
+          <SymbolIcon
+            symbol={token.displaySymbol}
+          />
+          <NumberFormat
+            value={vaultValue}
+            thousandSeparator
+            decimalScale={2}
+            displayType='text'
+            suffix='%'
+            renderText={(val: string) => (
+              <ThemedView
+                style={tailwind('px-1 items-center rounded')}
+              >
+                <ThemedText
+                  light={tailwind('text-gray-900')}
+                  dark={tailwind('text-gray-50')}
+                  style={tailwind('text-sm font-medium')}
+                  testID='bottom-sheet-vault-percentage-text'
+                >
+                  {val}
+                </ThemedText>
+              </ThemedView>
+            )}
+          />
+        </ThemedView>
+      </ScrollView>
+      <View style={tailwind('pt-2 flex justify-between flex-row')}>
+        <ThemedText style={tailwind('mr-2')}>{translate('components/AddOrRemoveCollateralForm', 'Resulting collateralization')}</ThemedText>
         <ThemedText
-          style={tailwind('mr-2')}
-        >{translate('components/AddOrRemoveCollateralForm', 'Resulting collateralization')}
-        </ThemedText>
-        <ThemedText
-          style={tailwind('font-semibold')}
+          style={tailwind('font-semibold pr-2')}
           light={hasInvalidColRatio ? tailwind('text-gray-300') : colors.light}
           dark={hasInvalidColRatio ? tailwind('text-gray-300') : colors.dark}
           testID='resulting_collateralization'
         >{hasInvalidColRatio ? translate('components/AddOrRemoveCollateralForm', 'N/A') : `${resultingColRatio.toFixed(2)}%`}
         </ThemedText>
-      </ScrollView>
+      </View>
```

Comment:
Why we removed scrollview?

Reply:
It's not removed, its shifted to line 291, where the vault % will be the last item that can be seen on the BottomSheetModal without scrolling